cmd/eval-dev-quality/cmd/evaluate.go

```diff
@@ -177,6 +177,7 @@ func (command *Evaluate) Execute(args []string) (err error) {
 
 	// Gather models.
 	modelsSelected := map[string]model.Model{}
+	providerForModel := map[model.Model]provider.Provider{}
```

Comment:
I would do a function on the Model to get its associated provider as all models need to have a provider in the first place. It is a required Parent -> Child dependency right?

Reply:
*sigh* provider already imports model and it would result in a cyclic dependency. I say there is something wrong with our structure if that is happening.

Reply:
@bauersimon Please move that to a Follow-Up